packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -97,30 +97,40 @@ const defaultErrorTypeForNormalize = "genericError";
  * @returns A valid Fluid Error with any provided annotations applied
  * @param error - The error to normalize
  * @param annotations - Annotations to apply to the normalized error
+ * @param messagePrefix - String to add to beginning of error message
  */
 export function normalizeError(
     error: unknown,
     annotations: IFluidErrorAnnotations = {},
+    messagePrefix?: string,
 ): IFluidErrorBase {
     // Back-compat, while IFluidErrorBase is rolled out
     if (isValidLegacyError(error)) {
         patchLegacyError(error);
     }
 
+    let existingProps: ITelemetryProperties | undefined;
     if (isFluidError(error)) {
-        // We can simply add the telemetry props to the error and return it
-        error.addTelemetryProperties(annotations.props ?? {});
-        return error;
+        if (!messagePrefix) {
```

Comment:
This doesn't feel right - We will lose errorType in this case, all because of the message prefix.  I will continue to look and suggest an alternative.